### PR TITLE
refactor: replace occurrences of password with passphrase

### PR DESCRIPTION
**Pull request by iccicci**
# Context

We need to replace "password" with "passphrase" for [BIP-39](https://github.com/bitcoin/bips/blob/master/bip-0039.mediawiki) compliance.

# Proposed Solution

Replace the relevant occurrences of `password` with `passphrase`.

### CODE REVIEW

File: packages/e2e/README.md [hunk 1]
```diff
@@ -158,7 +158,7 @@ STAKE_POOL_PROVIDER=stub
 STAKE_POOL_PROVIDER_PARAMS='{"baseUrl":"http://localhost:4000/stake-pool"}'
 ```
 
-> :information_source: Notice that KEY_MANAGEMENT_PARAMS *mnemonic* property is empty, if you leave this empty on the **local network's** e2e tests a new set of random mnemonics will be generated for you, this is the recommended way of setting up e2e tests on this network.
+> :information*source: Notice that KEY_MANAGEMENT_PARAMS \_mnemonic* property is empty, if you leave this empty on the **local network's** e2e tests a new set of random mnemonics will be generated for you, this is the recommended way of setting up e2e tests on this network.
```
Comment (rhyslbw) [inline packages/e2e/README.md:161]: Looks like a find/replace error?

image

**Review by rhyslbw — CHANGES_REQUESTED** [review]
> Cardano compliance.

It's actually for [BIP-39](https://github.com/bitcoin/bips/blob/master/bip-0039.mediawiki) compliance

File: packages/e2e/README.md (hunk 1 above)
Comment (iccicci) [reply packages/e2e/README.md]: This is due to some extensions in my VSCode which takes some not requested initiative. Sorry.

29e4a156d should solve this

**Review by mkazlauskas — DISMISSED** [review]
Looks good! :rocket:

**Review by rhyslbw — APPROVED** [review]
Ready for squashing

**Review by rhyslbw — APPROVED** [review]

**Pull request merged**